src/types/course.ts

```diff
@@ -79,6 +79,17 @@ export type ProjectProps = {
   ordine?: number;
   copertina: ImageDataLike;
   url?: string;
+  metaDescription?: string;
+  body: ProjectBodyProps;
+};
+
+export type ProjectBodyProps = {
+  body: string;
+  childMarkdownRemark: {
+    headings: object[];
+    html: string;
+    rawMarkdownBody: string;
+  };
```

Comment:
Se non vuoi impazzire con questa roba se aggiungiamo questo, possiamo aprire un issue dove inserendo questo in gatsby.config
```
module.exports = {
  graphqlTypegen: true,
}
```

Va a crearci un file con i tipe delle query che facciamo. Lo farei in un altra PR però

Reply:
Bella hit sta roba